dashboard/db/migrate/20211210183930_add_application_id_to_pd_enrollment.rb

```diff
@@ -0,0 +1,13 @@
+class AddApplicationIdToPdEnrollment < ActiveRecord::Migration[5.2]
+  def change
+    add_column :pd_enrollments, :application_id, :integer
+
+    Pd::Enrollment.where.not(user_id: nil) do |enrollment|
+      Pd::Application::ApplicationBase.where(user_id: enrollment.user_id).each do |application|
+        if application.try(:pd_workshop_id) == Pd::Enrollment.pd_workshop_id
```

Comment:
What happens if we leave this line off? What would we be missing?

Reply:
It's possible a user has more than one application (from different years). I could check that the year is current, but this behavior matches what we're currently doing in https://studio.code.org/pd/workshop_dashboard/workshops/9856, which is what was asked for